### PR TITLE
Add the AddThingsToDoAd example

### PR DESCRIPTION
**Pull request by Raibaz**

### CODE REVIEW

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+            CampaignBudgetServiceClient budgetService = client.GetService(
+                Services.V14.CampaignBudgetService);
+
+            // Create the campaign budget.
+            CampaignBudget budget = new CampaignBudget()
+            {
+                Name = "Interplanetary Cruise Budget #" + ExampleUtilities.GetRandomString(),
+                DeliveryMethod = BudgetDeliveryMethod.Standard,
+                AmountMicros = 500000
+            };
```
Comment (fiboknacky) [inline Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs:131]: You miss 'explicitly_shared' = true

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+                NetworkSettings = new NetworkSettings
+                {
+                    TargetGoogleSearch = true
+                },
+
+                // Optional: Set the start date.
+                StartDate = DateTime.Now.AddDays(1).ToString("yyyyMMdd"),
+
+                // Optional: Set the end date.
+                EndDate = DateTime.Now.AddYears(1).ToString("yyyyMMdd"),
```
Comment (fiboknacky) [inline Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs:192]: FYI, this is not in the golden CL.

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+            // [END add_things_to_do_ad_1]
+
+            MutateCampaignsResponse response = campaignService.MutateCampaigns(
+                customerId.ToString(), new CampaignOperation[] { new CampaignOperation() {
+                    Create = campaign
+                }}
+            );
+
+            string campaignResourceName = response.Results[0].ResourceName;
+            Console.WriteLine("Campaign with resource ID = '{0}' was added.", campaignResourceName);
```
Comment (fiboknacky) [inline Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs:203]: should be "...with resource name = ...."

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+            };
+
+            MutateAdGroupsResponse response = adGroupService.MutateAdGroups(
+                customerId.ToString(), new AdGroupOperation[] { new AdGroupOperation() {
+                    Create = adGroup
+                }}
+            );
+
+            string adGroupResourceName = response.Results[0].ResourceName;
+            Console.WriteLine("Ad group with resource ID = '{0}' was added.", adGroupResourceName);
```
Comment (fiboknacky) [inline Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs:238]: Ditto. A "name" not "ID".

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+            };
+
+            MutateAdGroupAdsResponse response = adGroupAdService.MutateAdGroupAds(
+                customerId.ToString(), new AdGroupAdOperation[] { new AdGroupAdOperation() {
+                    Create = adGroupAd
+                }}
+            );
+
+            string adGroupAdResourceName = response.Results[0].ResourceName;
+            Console.WriteLine("Ad group ad with resource ID = '{0}' was added.",
```
Comment (fiboknacky) [inline Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs:280]: Ditto

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+                adGroupAdResourceName);
+        }
```
Comment (fiboknacky) [inline Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs:282]: No "// [END add_things_to_do_ad_3]" found

**Review by fiboknacky — CHANGES_REQUESTED** [review]

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs (hunk 6 above)
Comment (Raibaz) [reply Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs:277]: Done.

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs (hunk 5 above)
Comment (Raibaz) [reply Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs]: Done.

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs (hunk 4 above)
Comment (Raibaz) [reply Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs]: Done.

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs (hunk 3 above)
Comment (Raibaz) [reply Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs]: Done.

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs (hunk 2 above)
Comment (Raibaz) [reply Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs]: Removed.

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs (hunk 1 above)
Comment (Raibaz) [reply Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs]: Done.

File: Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+using static Google.Ads.GoogleAds.V14.Enums.AdvertisingChannelSubTypeEnum.Types;
+using static Google.Ads.GoogleAds.V14.Enums.BudgetDeliveryMethodEnum.Types;
+using static Google.Ads.GoogleAds.V14.Enums.CampaignStatusEnum.Types;
+using static Google.Ads.GoogleAds.V14.Resources.Campaign.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V14
+{
+    /// <summary>
+    /// This example creates a Things to do campaign, an ad group and a Things to do ad.
+    /// Prerequisite: You need to have an access to the Things to Do Center. The integration
```
[the remark this answers is not in the document]
Comment (Raibaz) [reply Google.Ads.GoogleAds/examples/Travel/AddThingsToDoAd.cs]: Done.

**Review by jradcliff — APPROVED** [review]

**Pull request merged**